src/Clauses/OrderByClause.php

```diff
@@ -47,47 +49,36 @@ class OrderByClause extends Clause
     /**
      * Add a property to sort on.
      *
-     * @param Property $property The additional property to sort on
+     * @param Property $property The additional property to sort on.
+     * @param string|null $order The order of the property to appear. Null is equal to the default in Neo4J.
+     *
      * @return OrderByClause
      */
-    public function addProperty(Property $property): self
+    public function addProperty(Property $property, ?string $order = null): self
     {
-        $this->properties[] = $property;
+        $this->orderings[] = new Order($property, $order);
 
         return $this;
     }
 
     /**
      * Returns the properties to order.
      *
-     * @return Property[]
+     * @return Order[]
```

Comment:
It seems like it does not return Orders, rather it seems to be returning AnyType.

Reply:
It looks like it returns Orders to me?

Reply:
if `$this->orderings = [new Order($n->property('name'), 'ASC'), new Order($n->property('age'), 'DESC')]`, this function would return `[$orderings[0]->getExpression(), $orderings[1]->getExpression()]` = `[$n->property('name'),$n->property('age')]` if I'm not mistaken.

Reply:
Looks like I looked at the wrong lines of code! You are entirely correct. Great find